main.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import csv
+import NN
+import embedding as embd
+import json
+import pickle
+import utils as u
+import NN
+
+
+def main():
+    if len(sys.argv) != 2:
+        print("[config.json]")
+        return
+    parameters = u.load_file(sys.argv[1])
+    train_file, tokenization_type, language_model, embedding = u.get_embedding_parameters(parameters)
+    X_train, y_train = embd.get_x_y(tokenization_type, train_file, language_model, embedding)
```

Comment:
```suggestion
    X_train, y_train = embd.get_x_y(**parameters["embeddings"])
```

And then modify `embd.get_x_y` to accept `**kwargs` as parameters.

Similarly, you can change `model.train`. This way, the code will be more concise and `get_...` functions in utils will not be necessary.